packages/expo-age-range/src/AgeRange.ts

```diff
@@ -0,0 +1,13 @@
+import ExpoAgeRange from './ExpoAgeRange';
+import { AgeRangeRequest, AgeRangeResponse } from './ExpoAgeRange.types';
+
+/**
+ * Prompts user to declare their age range.
+ * @return A promise that resolves with user's age range response. Or rejects with an error (TODO vonovak export the error codes?).
+ * @platform ios 26.0+
+ * @platform android
+ */
+export async function requestAgeRangeAsync(options: AgeRangeRequest): Promise<AgeRangeResponse> {
+  // TODO vonovak can just export the function directly?
```

Comment:
Yeah, I think we don't have to wrap the native calls. Just make sure the docs data are generated correctly if you move the doc comment to `ExpoAgeRangeModule` interface.